fontforge/featurefile.c

```diff
@@ -2277,6 +2277,7 @@ struct parseState {
     unsigned int backedup: 1;
     unsigned int skipping: 1;
     unsigned int in_ufo: 1;
+    unsigned int ignore_invalid_sub: 1;      // Don't error out if a substitution rule is not applicable. Used for merging feature files
```

Comment:
Not understanding why this name was chosen. It's just as confusing. If you want `sub` mentioned put `sub_replacement`.

Reply:
May be  I should  use `ignore_invalid_gsub` because this flag actually handles GSUB rules. Does that work, or `ignore_invalid_replacement` as you suggested earlier? Either way is ok with me.

Reply:
Please let me know the preference; I shall update the pull request accordingly.

Reply:
`ignore_invalid_replacement`. Because there's only one other type of place this might make sense, on a `pos`.